### PR TITLE
DAT-229 feat(FMI): added FMI nodes and updated others

### PR DESCRIPTION
**Pull request by dmiller15**
r? @allisonheath @ZhenyuZ @msfitzsimons @dankolbman

### CODE REVIEW

**Comment by dmiller15** [comment]
Could you perhaps detail how the FoundationOne_Annotation_Workflow and FoundationOne_Variant_Aggregation_and_Masking_Workflow would look? I'm working on getting those other changes in right now.

File: gdcdictionary/schemas/structural_variation.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+id: "structural_variation"
+title: Structural Variation
+type: object
+namespace: http://gdc.nci.nih.gov
+category: data_file
+program: '*'
+project: '*'
+description: >
+  Data file containing the structural variation data from the FoundationOne Reports.
```
Comment (allisonheath) [inline gdcdictionary/schemas/structural_variation.yaml]: Shouldn't this be general and not include "from the FoundationOne Reports"

File: gdcdictionary/schemas/submitted_genomic_profile.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+id: "submitted_genomic_profile"
+title: Submitted Genomic Profile
+type: object
+namespace: http://gdc.nci.nih.gov
+category: data_file
+program: '*'
+project: '*'
+description: >
+  Data file containing genomic profile information from Foundation Medicine.
```
Comment (allisonheath) [inline gdcdictionary/schemas/submitted_genomic_profile.yaml]: Should be generic, not specific to FMI.

File: gdcdictionary/schemas/genomic_profile_harmonization_workflow.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  - name: submitted_genomic_profiles
+    backref: genomic_profile_harmonization_workflows
+    label: performed_on
+    target_type: submitted_genomic_profile
+    multiplicity: one_to_one
+    required: true
+
+required:
+  - submitter_id
+  - workflow_link
```
Comment (dmiller15) [inline gdcdictionary/schemas/genomic_profile_harmonization_workflow.yaml]: Workflow link refers to the link to the open source code used to run the workflow. So these are the github commits in our harmonization pipelines. Will that not be the case here?

Comment (allisonheath) [inline gdcdictionary/schemas/genomic_profile_harmonization_workflow.yaml]: Oh right - forgot about this property getting pulled in from `_terms.yaml#/workflow_type`

**Pull request merged**